External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -87,6 +89,10 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 	bLegacy2BIDS = true;
 end
 
+if nargin < 10 || isempty(bOverwrite)

```

Comment:
Make sure the length is three. Note for myself, I will fix this.
true -> "true true true"
false -> false false false"
true false -> true false true
false false -> false false true

Reply:
Sure though with length<3 I would also issue a warning, since they may not have thought about skipping JSON/MAT

Reply:
Good point. if length == 1, then we prolong and issue a warning. If length == 2, then we throw an error because it's actually a bit unclear if [1 0] should extend to [1 0 1] or [1 0 0] etc., so best to throw an error.

Done in: 39d293ee